session/pessimistic_test.go

```diff
@@ -1833,3 +1835,71 @@ func (s *testPessimisticSuite) TestAmendForUniqueIndex(c *C) {
 	tk.MustExec("commit")
 	tk2.MustExec("admin check table t")
 }
+
+func (s *testPessimisticSuite) TestResolveStalePessimisticPrimaryLock(c *C) {
```

Comment:
It's complex. I want to confirm Does it fail with the always cached version?

Reply:
I've  tested without cache change, it will fail with error reported from `admin check` statement.